docs/releasing/publishing.md

```diff
@@ -14,6 +14,8 @@
 3. Write a summary to post on Slack after the release. For example:
 
     🚀 We’ve just released GOV.‌UK Frontend v3.7.0. It's now easier and faster to use our Sass. We've also made improvements to back links, breadcrumbs, lists and the header. Thanks to @SLACK-NAME for helping with this release. [https://github.com/alphagov/govuk-frontend/releases/tag/v3.7.0](https://github.com/alphagov/govuk-frontend/releases/tag/v3.7.0)
+    Remember that Prototype Kit needs its own summary, since it has its own release notes.
```

Comment:
This needs an additional line above it to so it's separate from the example summary above.

Reply:
```suggestion

    Remember that Prototype Kit needs its own summary, since it has its own release notes.
```